Client/Frontend/UserContent/UserScripts/Playlist.js

```diff
@@ -0,0 +1,348 @@
+window.__firefox__.includeOnce("Playlist", function() {
+    
+    
+    function notify(target, type) {
+        if (target) {
+            var name = target.title;
+            if (name == null || typeof name == 'undefined' || name == "") {
+                name = document.title;
+            }
+            
+            if (target.src !== "") {
+                window.webkit.messageHandlers.playlistHelper.postMessage({
```

Comment:
You should add the `SECURITY_TOKEN` to the message handlers to avoid webpage scripts from invoking the message handlers directly.

Reply:
Fixed.